_consulting/organisational-leadership.md

```diff
@@ -55,4 +55,20 @@ Country-specific nuances, e.g. setting up a [stichting](https://en.wikipedia.org
 
 **Costs: £3000**
 
-LMIC discounts are available for all offers. If you are interested in any of the services, please contact us at team@we-are-ols.org. 
+##  Research Software Engineer (RSE) Leadership Training Course 
+* **Course description:**
+  * This leadership course is possibly the first in the world to provide an **intersectional framework** that underpins a leadership course for those in the tech industry.
+  * It focuses on centering the individual and prioritising based on the "degree of difficulty" across recruitment, retention, and relationship management.
+  * This course lays bare the core, practical concepts that you need to know to be an outstanding leader. 
```

Comment:
```suggestion
  * This course lays bare the core, practical concepts that you need to know to be an outstanding leader for a team of 1+ staff in an organisation. 
```

To differentiate fro mteh open seeds program, which is about open community leaderdship.

Reply:
let's hope "how to write without incredible typos" isn't in the course, I certainly am not capable :P